techmatters/webchat
# Emoji Picker in Webchat

## Description
This PR adds Emoji Picker to webchat. 

### Checklist
- [n/a] New tests added



### Related Issues
Fixes # [1640](https://tech-matters.atlassian.net/browse/CHI-1640)

### Verification steps
- run a clean install - `npm ci`. Build and run the widget
- To check use of mobile optimization, use chrome dev tools to check different resolutions. 
- To check localization, hard code a language in `EmojiPicker.tsx` to any other language strign used in configuration. I.e., to check french, hardcode `fr-CA` and check that the picker is localized to French
- To check blocked emojis, add or remove the emoji in `blockedEmojis.ts` and check the Picker in the widget.

![Screenshot 2023-03-02 at 1 41 27 PM](https://user-images.githubusercontent.com/102122005/222522041-f7b08fac-fb4d-48f6-8537-b34fa70b918e.png)
![Screenshot 2023-03-02 at 1 41 20 PM](https://user-images.githubusercontent.com/102122005/222522073-bf56663a-0aef-415e-984c-7b5c09e2a7b8.png)
